CONTRIBUTING.md

```diff
@@ -3,7 +3,7 @@
 #### **Did you find a bug?**
 
 * **Do not open up a GitHub issue if the bug is a security vulnerability
-  in Rails**, and instead to refer to our [security policy](http://rubyonrails.org/security/).
+  in Rails**, and instead to refer to our [security policy](https://rubyonrails.org/security/).
```

Comment:
rubyonrails.org is http-only currently.

Reply:
@jeremy Weird, I swear the HTTPS link worked a few hours ago.

Reply:
We're hosted with GitHub pages, so when/if they support https on apex domain, the link will work! Thanks for the PR in any case 😊

Reply:
Don't know if this has been answered before, but... is there any plan (or has been rejected) to use Cloudflare as an intermediate to get the SSL done?